include/ibamr/IBHydrodynamicSurfaceForceEvaluator.h

```diff
@@ -103,9 +102,40 @@ class IBHydrodynamicSurfaceForceEvaluator
     virtual ~IBHydrodynamicSurfaceForceEvaluator();
 
     /*!
-     * \brief Compute the hydrodynamic force via surface integration.
+     * \brief Compute the hydrodynamic force via surface integration after all the hierarchy
+     * variables have been integrated.
+     *
+     * \note The users should call this routine if the hydrodynamic forces need to be computed just
+     * as a postprocessing step.
+     */
+    virtual void computeHydrodynamicForceTorque(IBTK::Vector3d& pressure_force,
+                                                IBTK::Vector3d& viscous_force,
+                                                IBTK::Vector3d& pressure_torque,
+                                                IBTK::Vector3d& viscous_torque,
+                                                const IBTK::Vector3d& X0);
+
+    /*!
+     * \brief Compute the hydrodynamic force via surface integration while the integration of
+     * variables is happening during a given timestep.
+     */
+    virtual void computeHydrodynamicForceTorque(IBTK::Vector3d& pressure_force,
+                                                IBTK::Vector3d& viscous_force,
+                                                IBTK::Vector3d& pressure_torque,
+                                                IBTK::Vector3d& viscous_torque,
+                                                const IBTK::Vector3d& X0,
+                                                double time,
+                                                double current_time,
+                                                double new_time);
+    /*!
+     * \brief Set the surface contour value <em> s <\em> of the level set field, such that \f$ \phi(s) = \partial \Omega
+     * \f$.
      */
-    virtual void computeHydrodynamicForce();
```

Comment:
We should at least log that we broke this interface under 'incompatible changes' and fix any relevant examples to not use it.